test/utils/update/managing.officer.mapper.spec.ts

```diff
@@ -25,6 +25,7 @@ describe("Test mapping to managing officer", () => {
   test('map officer data to managing officer should return object', () => {
     expect(mapToManagingOfficer(managingOfficerMock)).toEqual({
       id: undefined,
+      ch_reference: "",
```

Comment:
undefined as I don't expect it to be ""

Reply:
It is "". Mock data below
 links: {
    officer: {
      appointments: ""
    }
  },

Reply:
will need to be 
links: { self: "selflink" }
The undefined / "" is a minor concern